src/Sparrow.Server/Platform/RavenProcess.cs

```diff
@@ -174,7 +180,6 @@ private Task<string> ReadLineAsync(FileStream fs, CancellationToken ctk)
             {
                 if (sb == null)
                     sb = new StringBuilder();
-
                 var c = Encoding.UTF8.GetString(buffer, 0, read);
 
                 if (buffer[0] == '\n')
```

Comment:
This is wrong, the `\n` can happen anywhere in the buffer, not just the first one.

Reply:
This is 1 byte buffer.  However I am going to rewrite this part (read 4096 and store the reset after \n for next iteration)